### PR TITLE
Tests are now being run by tox

### PR DESCRIPTION
**Pull request by chartjes**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[tox]
+envlist = py3
```
Comment (Natim) [inline tox.ini]: envlist = py36,flake8

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+skipsdist = true
+
+[testenv]
+passenv = * 
+deps = -rrequirements.txt
+commands = pytest --env={env:TEST_ENV:} {posargs: config-tests} -s
+
+[testenv:flake8]
+skip_install = true
+deps = -rdev-requirements
```
Comment (Natim) [inline tox.ini]: why not just `flake8` there?

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+envlist = py3
+skipsdist = true
+
+[testenv]
+passenv = * 
+deps = -rrequirements.txt
+commands = pytest --env={env:TEST_ENV:} {posargs: config-tests} -s
+
+[testenv:flake8]
+skip_install = true
```
Comment (Natim) [inline tox.ini:11]: 👍

File: tox.ini [hunk 4]
```diff
@@ -8,9 +8,9 @@ deps = -rrequirements.txt
 commands = pytest --env={env:TEST_ENV:} {posargs: config-tests} -s
 
 [testenv:flake8]
-skip_install = true
-deps = -rdev-requirements
-commands = flake8 {posargs:.}
+deps = -rrequirements.txt
+commands = flake8 {posargs: config-tests/}
+whitelist_externals = flake8
```
Comment (Natim) [inline tox.ini]: We should install flake8 in the tox venv.

Something like: 
```
[testenv:flake8]
skip_install = true
deps = flake8
commands = flake8 {posargs: config-tests/}
```

File: config-tests/test_webextensions_crud.py [hunk 5]
```diff
@@ -39,26 +39,29 @@ def test_add_content(env, conf):
...
     # Add a record to our QA collection and make sure we have N+1 records
-    existing_records = client.get_records(collection=conf.get(env, 'qa_collection'), bucket='default')
+    existing_records = client.get_records(
+        collection=conf.get(env, 'qa_collection'), bucket='default')
     assert len(existing_records) == 0
 
     data = {"payload": {"encrypted": "SmluZ28gdGVzdA=="}}
-    resp = client.create_record(data, collection=conf.get(env, 'qa_collection'), bucket='default')
+    resp = client.create_record(
+        data,
```
Comment (Natim) [inline config-tests/test_webextensions_crud.py:48]: nit: Maybe we can keep data on the previous line here?

**Review by Natim — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 6]
```diff
@@ -5,4 +5,4 @@ WORKDIR /app
 RUN pip3 install -r requirements.txt
 RUN find . -name "*.pyc" | xargs rm -rf
```
Comment (rpappalax) [inline Dockerfile]: it's probably better practice to put this cleanup in a .dockerignore (one less layer)

**Comment by rpappalax** [comment]
+1 on my end

**Pull request closed without merging**